bitnami/mysql/templates/primary/configmap.yaml

```diff
@@ -9,6 +9,9 @@ metadata:
     {{- if .Values.commonLabels }}
     {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
     {{- end }}
+    {{- if .Values.primary.podLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.primary.podLabels "context" $ ) | nindent 4 }}
+    {{- end }}
```

Comment:
Why are the podLabels needed in the configmap labels?

Reply:
Yeah I think it can be omitted from configMap will make changes for primary and secondary both.